integration/config/core.yaml

```diff
@@ -161,6 +165,10 @@ peer:
         aliveExpirationTimeout: 25s
         # Reconnect interval(unit: second)
         reconnectInterval: 25s
+        # Max number of attempts to connect to a peer
+        maxConnectionAttempts: 120
```

Comment:
if things do not work, we would have to wait 120 attempts \times 25 seconds = 50min for the peer(?) to fail.
I guess 2-3 should be enough.

Reply:
this is the default from sampleconfig. Unless we have a good reason to change it, i would leave default configs as-is ...